sdk/tables/Azure.Data.Tables/src/TableClientOptions.cs

```diff
@@ -30,10 +30,17 @@ public TableClientOptions(ServiceVersion serviceVersion = Latest)
             VersionString = serviceVersion switch
             {
                 ServiceVersion.V2019_02_02 => "2019-02-02",
+                ServiceVersion.V2020_12_06 => "2020-12-06",
```

Comment:
Is there a new swagger version to go along with this?

Reply:
No :)